ResearchKit/Common/ORKObserver.h

```diff
@@ -0,0 +1,49 @@
+//
+//  ORKObserver.h
+//  ResearchKit
+//
+//  Created by Ricardo Sánchez-Sáez on 22/04/2015.
+//  Copyright (c) 2015 researchkit.org. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@class ORKTaskViewController;
+
+@interface ORKObserver : NSObject
+
+@property (nonatomic, strong) NSArray *keyPaths;
+@property (nonatomic, strong) id target;
+@property (nonatomic) BOOL observing;
+@property (nonatomic) void *context;
+
+@property (nonatomic, weak) id responder;
+@property (nonatomic) SEL action;
+
+- (void)startObserving;
+
+@end
+
+
+@protocol ORKViewControllerObserverProtocol <NSObject>
+@required
+- (void)collectToolbarItemsFromViewController:(UIViewController *)viewController;
+@end
+
+@interface ORKViewControllerObserver : ORKObserver
```

Comment:
I don't think anything else will. Let's just move that back. It's just a subclass of ORKObserver, so it's fine to have it internal to the task VC.